CHANGELOG.md

```diff
@@ -44,6 +44,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Relax visibility of the HTTP_CHANNEL_KEY and HTTP_SERVER_CHANNEL_KEY to make it possible for the plugins to access associated Netty4HttpChannel / Netty4HttpServerChannel instance ([#4638](https://github.com/opensearch-project/OpenSearch/pull/4638))
 - Use ReplicationFailedException instead of OpensearchException in ReplicationTarget ([#4725](https://github.com/opensearch-project/OpenSearch/pull/4725))
 - Migrate client transports to Apache HttpClient / Core 5.x ([#4459](https://github.com/opensearch-project/OpenSearch/pull/4459))
+- Refactored BalancedAllocator.Balancer to LocalShardsBalancer ([#4761](https://github.com/opensearch-project/OpenSearch/pull/4761))
```

Comment:
Please take a look at the updated [changelog process](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog).

A couple points here:
- there are a few extra lines from other changes
- maybe this PR is part of a bigger feature (e.g. "Add operation-level durability to remote-backed indexes") that should be a single changelog entry?
- assuming this is intended to be backported, then it should go in the 2.x section of the changelog

Reply:
Thanks @andrross. This definitely makes sense.

Reply:
This qualifies for adding the `skip-changelog` label. However, I don't have sufficient access to add labels or even reviewers for that matter. Is there any process or guidelines for the same?